server/index.js

```diff
@@ -0,0 +1,94 @@
+const app = require('express')();
+const server = require('http').Server(app)
+const io = require('socket.io')(server);
+const port = 3000;
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const methods = require('./helpers/helperFuncs');
+
+app.use(bodyParser.json());
+app.use(cors());
+
+let activeRooms = [];
+let appUserCount = 0;
+
+io.on('connection', (socket) => {
```

Comment:
It can be a problem to have everything in one single file, but given the size it's ok. If this file starts to grow, you can handle socket events the same way as a restful api. A "router" that listen to events and tunnel them to a controller, which talks to the models.

Reply:
@arol just reviewed all of your comments. I will adjust in my first refactor of the app! Thanks for all of your help!!